src/test/groovy/life/qbic/domain/sample/SampleEventStoreSpec.groovy

```diff
@@ -33,5 +33,10 @@ class SampleEventStoreSpec extends Specification {
     List<SampleEvent> findAllForSample(SampleCode sampleCode) {
       return events
     }
+
+    @Override
+    List<SampleEvent> findAllMatchingProject(ProjectCode projectCode) {
+      return null
```

Comment:
Shouldn't this also return events for testing purposes?

Reply:
no, we have no tests using this 😄